tests/cases/standalone/common/flow/flow_tql.result

```diff
@@ -91,7 +91,7 @@ CREATE TABLE http_requests (
 
 Affected Rows: 0
 
-CREATE FLOW calc_reqs SINK TO cnt_reqs AS
+CREATE FLOW calc_reqs SINK TO cnt_reqs EVAL INTERVAL '1m' AS
 TQL EVAL (0, 15, '5s') count_values("status_code", http_requests);
```

Comment:
About the grammar:
- we should check and warn/reject the absolute start/end (not express with `now()`)
- I don't prefer to add a new grammar section for this, and 
- it's only for batching mode
- How about using `step` as the interval?

Reply:
> About the grammar:
> 
>     * we should check and warn/reject the absolute start/end (not express with `now()`)
> 
>     * I don't prefer to add a new grammar section for this, and
> 
>     * it's only for batching mode
> 
>     * How about using `step` as the interval?

step still sound too frequency, how about using `end - start`?